prime-router/docs/onboarding-users/receivers.md

```diff
@@ -1,12 +1,307 @@
+     - Patient Zip Code
+     - Patient Phone
+     - Patient Email
+    
+- At least one of the following date fields has a valid date:
+     - Test Ordered Date
+     - Specimen Collection Date
+     - Test Result Date
+    
+- The following fields are 10 characters in length:
+     - Testing Lab CLIA
+     - Ordering Facility CLIA
+    
+- Processing Code does not equal T nor D
+
+- Only test results that pass all the above requirements will be transferred to the jurisdiction. There are two options 
+for jurisdictions that want all results reported.
+- Use AllowAll() to bypass all quality filters 
+- Create a secondary feed with the reverseQualityFilter() set to true. This will only allow results that fail the 
+quality filters listed above.
+
+The mechanism for how each record is translated is laid out in the schema, which is discussed in the next section.
+
+### 2. Set up a new schema
+*NOTE - IF YOU ARE WORKING ON AN HL7 RECEIVER, YOU DO NOT NEED TO CREATE A NEW SCHEMA.*
+
+* By default, any HL7 receiver will use the universal or covid schema and you do not need to create a schema
+specific to your receiver.
+* If they are going to receive a CSV or FHIR file you *MUST* create a schema. In lieu
+of a schema, we use the `TranslationConfig` to set default values and control HL7 processing. 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how_to_use_translation_configuration_features.md)
+* For additional information on creating a schema see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/how-to-onboard-a-sender.md)
+
+### 3. Generate test data
+* Generate fake, or better, synthesized test data. Prime has two ways to generate anonymous fake data (Fake or Syntheic data):
+1. Fake data - Fake data uses a library to generate purely fake data for ALL data points except for the city, state, 
+postal code, and county, which are tied to actual locations. The data that is generated is somewhat constrained to 
+resemble reasonable defaults, but is designed to be very random, which allows us to test the limits of validation 
+systems. *THERE IS NO PII OR PHI IN FAKE DATA*
+2. Synthetic data - Synthetic data takes a file of actual clinical results and does a combination of shuffling some 
+PHI/PII and faking other data points, so the records cannot be traced back to the patient, but the actual portion of 
+positive to negative tests, lab names & CLIA's, names of ordering providers, etc will be actual valid information. The 
+goal is to provide a higher-quality, less-random, dataset that can then be used to validate the information being sent 
+from PRIME to the receivers. *While great care has been taken to ensure we do not leak PII/PHI, this should not be used 
+except with receivers we are in the process of onboarding.*
+
+#### Generate some fake data for testing
+```
+./prime data --input-fake 100 --input-schema lt/lt-covid-19 --output myfile.csv --target-states LT
```

Comment:
I'm not sure how this command works, but to work for the UP. It would have to generate FHIR or HL7 and it shouldn't need to pass a schema. Can it do that? If not we should probably remove this section and maybe create a ticket to look into adding that for the UP.

Reply:
I removed it.